min/lib/Minify/Controller/MinApp.php

```diff
@@ -136,13 +136,14 @@ public function setupSources($options) {
                 $base = '/';
             }
             $allowDirs = array();
+            $document_root = realpath($_SERVER['DOCUMENT_ROOT']);
```

Comment:
I'm sorry, master branch looks quite differently. $_SERVER['DOCUMENT_ROOT'] is used in many places and it's hard to figure out whether issue still exists.
The problem was that file names were realpathed, but comparison was with not realpathed DOCUMENT_ROOT
I appreciate if you could pay attention to this problem